Makefile

```diff
@@ -8,22 +8,28 @@ BUILD_ID ?= $(shell git describe --tags)
 # len("alpine-lima-12345-3.13.5-x86_64") == 31
 EDITION ?= std
 
+ARCH ?= $(shell uname -m)
+ARCH_ALIAS_x86_64 = amd64
+ARCH_ALIAS_aarch64 = arm64
+
 NERDCTL_VERSION=0.12.1
 
 .PHONY: mkimage
 mkimage:
 	cd src/aports && git fetch && git checkout $(GIT_TAG)
 	docker build \
-		--tag mkimage:$(ALPINE_VERSION) \
+		--tag mkimage:$(ALPINE_VERSION)-$(ARCH) \
 		--build-arg ALPINE_VERSION=$(ALPINE_VERSION) \
+		--platform linux/$(ARCH) \
```

Comment:
The `--platform` value has to be like `linux/amd64`, so `uname -m` value such as `x86_64` can't be passed here

Reply:
Can we test `make`-ing the iso using GitHub Actions, at least for `amd64`?

Reply:
> The --platform value has to be like linux/amd64, so uname -m value such as x86_64 can't be passed here

docker actually support `linux/x86_64`, linux/amd64`, `linux/aarch64` and `linux/arm64` from my tests. But I will make it consistent just to be on the safe side.

> Can we test make-ing the iso using GitHub Actions, at least for amd64?

I am not sure how to trigger this

Reply:
> Can we test `make`-ing the iso using GitHub Actions, at least for `amd64`?

This should be a separate PR (see #10).